browser-test/src/admin/admin_predicates_expanded.test.ts

```diff
@@ -257,172 +315,214 @@ test.describe('create and edit predicates', {tag: ['@northstar']}, () => {
       await adminPredicates.expectCondition(1)
     })
 
-    await test.step('Choosing a question updates value options', async () => {
-      await adminPredicates.selectQuestion(
-        /* conditionId= */ 1,
-        /* subconditionId= */ 1,
-        questionText,
-      )
-      await expect(
-        page
-          .getByLabel('Value(s)', {id: 'condition-1-subcondition-1-value'})
-          .locator(`option[value="Seattle"]`),
-      ).not.toHaveAttribute('hidden')
-
-      await validateScreenshot(
-        page.getByTestId('condition-1'),
-        'values-with-address-question-selected',
-      )
-    })
-  })
-
-  test(`Create number predicate`, async ({
-    page,
-    adminQuestions,
-    adminPrograms,
-    adminPredicates,
-  }) => {
-    await loginAsAdmin(page)
-
-    const programName =
-      'Create and edit an eligibility predicate with number question'
-    const questionText = 'how many burritos have you eaten today?'
+    // Test single-value operator on applicable question types.
+    for (const questionType of [
+      QuestionType.CURRENCY,
+      QuestionType.DATE,
+      QuestionType.NUMBER,
+    ]) {
+      await test.step(`Select ${questionType.valueOf().toLowerCase()} question and validate single-value operator behavior`, async () => {
```

Comment:
```suggestion
      await test.step(`Select ${questionType} question and validate single-value operator behavior`, async () => {
```

(note) Just use the enum value. You don't need to bother with the valueOf parts